src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
ah ok, so based on the original issue i'd call it a "SketchSymbol", but any name other than "Symbol" is acceptable :-)